pkg/controller/restore_controller_test.go

```diff
@@ -18,11 +18,14 @@ package controller
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"io/ioutil"
 	"testing"
 	"time"
 
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Comment:
double-import from L-40.
Can be moved to L-40

Reply:
Fixed.